ex3.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+
+def find_rows_and_cols_of_matrix(matrix):
+    """
+    Functions gets a matrix and return the amount of rows and cols.
+    :param matrix:
+    :return:(tuple) rows and cols of the matrix.
+    """
+    return np.shape(matrix)
+
+
+if __name__ == '__main__':
+    print(find_rows_and_cols_of_matrix([1, 2, 3, 4, 5, 6, 7]))
+    print(find_rows_and_cols_of_matrix([[1, 2], [3, 4], [5, 6]]))
+    print(find_rows_and_cols_of_matrix([[1, 2, 3], [4, 5, 6], [7, 8, 9]]))
```

Comment:
File should end with a newline - 
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline